postgres-matrix.yml

```
@@ -5,4 +5,6 @@ version_matrix:
   - 2.16:
       postgres_versions: [ 10, 11, 12, 13, 14 ]
   - 2.17:
-      postgres_versions: [ 10, 11, 12, 13, 14, 15 ]
+      postgres_versions: [ 10, 11, 12, 13, 14, 15, 16 ]
```

It does not make sense to add PG16 support to 2.17. 2.18 should be the first version to support this version of PostgreSQL.

```suggestion
      postgres_versions: [ 10, 11, 12, 13, 14, 15 ]
```